### ISSUE TITLE
Moving ayypex.json

### ISSUE THREAD

**Issue by skullcheat** [EquinoxAlpha/ayypex#1]
How to move ayypex.json to /usr/share/vulkan/implicit_layer.d

https://i.postimg.cc/QCTXkhzq/Capture-d-cran-du-2023-04-30-12-30-22.png

**Comment by skullcheat** [comment]
Now I get several errors when I try to compile with make
https://i.postimg.cc/59sNc9Lj/Capture-d-cran-du-2023-04-30-12-43-07.png

**Comment by EquinoxAlpha** [comment]
About moving ayypex.json: Use the terminal with root privileges.

About the compile errors: It seems that I forgot to include the "lib" directory in the git repository, sorry about that. For now, just create the lib directory yourself.

**Comment by skullcheat** [comment]
compiled file thanks you very very much =D
how to change settings or open menu ( if have )
because all done but dont work 
https://i.postimg.cc/c4wPNX8C/Capture-d-cran-du-2023-04-30-13-30-21.png

**Comment by EquinoxAlpha** [comment]
It doesn't have a menu or settings yet as the README says, I'm planning on including it though
You can just go into firing range and it should work, you should see the dummies through walls.

**Comment by skullcheat** [comment]
steam vulkan shader need installed ? because it still doesn't work =(

**Comment by EquinoxAlpha** [comment]
That's weird, check that ayypex.json is where it is supposed to be. Also check that libayypex.so is actually in /usr/lib/ayypex. If the files are placed correctly, check the maps "file" of Apex /proc/$(pgrep R5Apex)/maps and see if libayypex.so is present anywhere within that file. If not, start steam through a terminal (command: "steam-runtime"), start Apex and see if you find anything related to libayypex.so in the log.

**Comment by skullcheat** [comment]
/usr/lib/ayypex done : https://i.postimg.cc/tThnzP3R/Capture-d-cran-du-2023-05-02-12-46-33.png
i copy/paste this command cat /proc/$(pgrep R5Apex)/maps | grep "libayypex.so"
but have nothing ? : https://i.postimg.cc/gcH6XVcN/Capture-d-cran-du-2023-05-02-12-48-47.png

**Comment by EquinoxAlpha** [comment]
No. Run "steam-runtime" in a terminal, then start Apex and look at what is in the terminal.

>/proc/$(pgrep R5Apex)/maps outputs an error
You need to run Apex first. Remember to run the command as root.

I'd advise you to learn some basic things about Linux first.
